ou_dedetai/installer.py

```diff
@@ -211,62 +194,24 @@ def ensure_wineprefix_init(app: App):
     logging.debug(f"> {init_file} exists?: {init_file.is_file()}")
 
 
-def ensure_winetricks_applied(app: App):
+def ensure_wineprefix_config(app: App):
     app.installer_step_count += 1
     ensure_wineprefix_init(app=app)
     app.installer_step += 1
-    app.status("Ensuring winetricks & other settings are applied…")
-    logging.debug('- disable winemenubuilder')
-    logging.debug('- settings renderer=gdi')
-    logging.debug('- corefonts')
-    logging.debug('- tahoma')
-    logging.debug('- settings fontsmooth=rgb')
-    logging.debug('- d3dcompiler_47')
-
-    if not app.conf.skip_winetricks:
-        usr_reg = None
-        sys_reg = None
-        usr_reg = Path(f"{app.conf.wine_prefix}/user.reg")
-        sys_reg = Path(f"{app.conf.wine_prefix}/system.reg")
-
-        # FIXME: consider supplying progresses to these sub-steps
-
-        if not utils.grep(r'"winemenubuilder.exe"=""', usr_reg):
-            app.status("Disabling winemenubuilder…")
-            wine.disable_winemenubuilder(app, app.conf.wine64_binary)
-
-        if not utils.grep(r'"renderer"="gdi"', usr_reg):
-            app.status("Setting Renderer to GDI…")
-            wine.set_renderer(app, "gdi")
-
-        if not utils.grep(r'"FontSmoothingType"=dword:00000002', usr_reg):
-            app.status("Setting Font Smoothing to RGB…")
-            wine.install_font_smoothing(app)
```

Comment:
Is this done without winetricks? Don't recall a line doing this.

Reply:
Before this PR we were doing it with winetricks, yes. The [now unused] function still exists in `wine.py` L398-401. But if we decide we want to keep this modification we can do it directly with a registry change rather than using winetricks. This is the same kind of change I made to `set_renderer_to_gdi` in `wine.py` L280-287.

Reply:
I'd like to still do it, I suspect we initially did it for a reason, would rather have a couple lines of python run than the installer get less stable